tests/common/plugins/loganalyzer/loganalyzer.py

```diff
@@ -17,15 +17,14 @@
 COMMON_EXPECT = join(split(__file__)[0], "loganalyzer_common_expect.txt")
 SYSLOG_TMP_FOLDER = "/tmp/syslog"
 
-
```

Comment:
I think two blank lines before class definition is in compliant with PIP8.

Reply:
Fixed.